### PR TITLE
Convert OpenXrFbFaceTrackingExtensionWrapper to drive an XRFaceTracker.

### PR DESCRIPTION
**Pull request by Malcolmnixon**
This PR depends upon https://github.com/godotengine/godot/pull/88312 to provide the XRFaceTracker and XRFaceModifier3D types. Until they are available and merged in, building the code requires:
1. Building Godot with the XRFaceTracker and XRFaceModifier3D nodes.
2. Running Godot with `--dump-extension-api` an overwriting the godot_cpp `extension_api.json` file
3. Building godot_openxr_vendors

The following video demonstrates the results:

https://github.com/GodotVR/godot_openxr_vendors/assets/1863707/576c7792-4a4d-49cd-8efa-f570c7eac846

### CODE REVIEW

File: demo/project.godot [hunk 1]
```diff
@@ -12,7 +12,7 @@ config_version=5
 
 config/name="OpenXR Demo"
 run/main_scene="res://main.tscn"
-config/features=PackedStringArray("4.2", "GL Compatibility")
+config/features=PackedStringArray("4.3", "GL Compatibility")
```
Comment (m4gr3d) [inline demo/project.godot:15]: Can you also bump the minimum version in `plugin.gdextension`

Comment (Malcolmnixon) [reply demo/project.godot:15]: Ah yes, nice catch. Done!

**Comment by Malcolmnixon** [comment]
This PR has been updated to use XR_FB_face_tracking2 which adds basic support for tongue position.

**Review by m4gr3d — APPROVED** [review]

**Comment by Malcolmnixon** [comment]
Rebased to give access to the new Godot 4.3-dev.3 types.

**Pull request merged**